src/dbus/client/thread_api_dbus.hpp

```diff
@@ -769,6 +769,20 @@ class ThreadApiDBus
      */
     ClientError GetSrpServerInfo(SrpServerInfo &aSrpServerInfo);
 
+#if OTBR_ENABLE_TREL
+    /**
+     * This method gets the TREL information.
+     *
+     * @param[out] aTrelInfo  The TREL information.
+     *
+     * @retval ERROR_NONE  Successfully performed the dbus function call
+     * @retval ERROR_DBUS  dbus encode/decode error
+     * @retval ...         OpenThread defined error value otherwise
+     *
+     */
+    ClientError GetTrelInfo(TrelInfo &aTrelInfo);
```

Comment:
I personally think it's not very useful to add client APIs because there're no existing use cases. I'm fine with adding it though.

Reply:
Let's add it like we do for other features.